crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -415,6 +430,41 @@ impl BuildSpecCommand {
 			true
 		};
 
+		// Prompt the user for deterministic build only if the profile is Production.
+		let deterministic = deterministic || (profile == Profile::Production && cli
+			.confirm("Would you like to build the Wasm runtime deterministically? (Recommended for production builds)")
```

Comment:
```suggestion
			.confirm("Would you like to build the runtime deterministically? (Recommended for production builds)")
```
Ignore if you think the original one is better.

Reply:
Makes sense, https://github.com/r0gue-io/pop-cli/commit/f6b4beb51fd255ffd30aa1865b9c77f05336c790